package.json

```diff
@@ -2,9 +2,25 @@
   "name": "cross-domain-safe-weakmap",
   "version": "1.0.10",
   "description": "Cross domain safe WeakMap.",
-  "main": "index.js",
+  "main": "dist/cross-domain-safe-weakmap.js",
+  "module": "dist/module",
   "scripts": {
-    "test": "gulp test"
+    "build": "npm run babel & npm run webpack && npm run flow:build",
```

Comment:
Also using ` & ` doesn't seem to work very well for me -- kicks the task into the background, so it's not great if we're using these npm commands in any shell scripts that need to wait for the return codes.